lib/rucio/web/rest/flaskapi/v1/replicas.py

```diff
@@ -134,12 +134,12 @@ def generate(vo):
                         yield f'  <glfn name="/atlas/rucio/{rfile["scope"]}:{rfile["name"]}">'
                         yield '</glfn>\n'
 
-                        idx = 0
+                        idx = 1
                         for replica in replicas:
```

Comment:
May I suggest `for idx, replica in enumerate(replicas, start=1)`?

Reply:
of course